core/server/namespace.go

```diff
@@ -17,45 +16,24 @@ const (
 var ErrNamespaceNotFound = errors.New("namespace not found")
 
 func (as *coreServer) GetFluxNamespace(ctx context.Context, msg *pb.GetFluxNamespaceRequest) (*pb.GetFluxNamespaceResponse, error) {
```

Comment:
while you are there, can you do me a massive favour and make `as` `cs` in this file 😁  pleeeeasseee

Reply:
done